airbyte-integrations/connectors/source-postgres/src/main/java/io/airbyte/integrations/source/postgres/PostgresSource.java

```diff
@@ -182,8 +177,7 @@ public String toJDBCQueryParams(final Map<String, String> sslParams) {
             .map((entry) -> {
               try {
                 final String result = switch (entry.getKey()) {
-                  case JdbcSSLConnectionUtils.SSL_MODE -> PARAM_SSLMODE + EQUALS + toSslJdbcParam(SslMode.valueOf(entry.getValue()))
-                      + JdbcUtils.AMPERSAND + PARAM_SSL + EQUALS + (entry.getValue() == DISABLE ? PARAM_SSL_FALSE : PARAM_SSL_TRUE);
```

Comment:
Does it work without PARAM_SSL?
I think it should but to verify that it's not needed on either disabled or enabled modes

Reply:
Just manually verified with a successful sync without SSL.

Reply:
Why exactly are we removing this again?

Reply:
The toggle is logically redundant.
Setting ssl mode to disabled has the same effect of setting this toggle to false.